data_structures/src/superblock.rs

```diff
@@ -450,6 +462,28 @@ impl SuperBlockState {
                     signing_committee
                 };
 
+            // Override superblock signing committee during the bootstrapping of wit/2
+            self.signing_committee = match get_environment() {
+                Environment::Mainnet => {
+                    // wit2 activates at 2922240 + 14 x 1920 + 7 (TAPI delay) x 1920 (activation delay)
+                    // disabling the activation committee at 299_712 implies we have 8 weeks time (8 x 4320)
+                    // to inplement a decentralized superblock committee selection mechanism
+                    if (292_224..299_712).contains(&superblock_index) {
```

Comment:
Unless I got it wrong, this is assuming that:
- The start of WIP-0028's first TAPI signaling period is known.
- The TAPI signaling will pass at first try, i.e. during the first signaling period.
- V2_0 will be scheduled from the very first epoch of V1_8?

Reply:
But even if there is a delay anywhere, that delay is basically subtracted from the 8 weeks extra time we give ourselves (and technically the subtraction is not even fully proportional to the delay because the 8 weeks is calculated using an epoch of 20s while a delay would imply 45s epochs).